cmd/jobservice/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/G-Research/armada/internal/common"
+	"github.com/G-Research/armada/internal/jobservice"
+	"github.com/G-Research/armada/internal/jobservice/configuration"
+)
+
+const CustomConfigLocation string = "config"
+
+func init() {
```

Comment:
I suggest looking at how the testsuite is implemented. I think that's a cleaner approach. It doesn't require having an init function and so on.